grass7/imagery/i.gabor/i.gabor.html

```diff
@@ -0,0 +1,148 @@
+<h2>DESCRIPTION</h2>
+
+<p>
+<em>i.gabor</em> computes texture raster maps from a user-specified Gabor
+filter bank and raster map.
+The 2 dimensional Gabor filter is an orientation sensitive filter which mimics
+the cells within the human visual cortex.
+The module can either produce individual rasters with the suffix of
+<b>windowsize_orientation_wavelength_offset_aspect</b> for each filter in the
+bank or a single combined (<b>-c</b>) raster outputting with only the basename.
+
+<p>
+The default orientations computed will be 0 (East/West), 45
+(North-East/South-West), 90 North/South, 135 (North-West/South-East).
+However, the user can also specify any number of orientations in degrees less
+than 180 with the <b>orientation</b> option.
+The window size of the Gabor filter is specified by the user in
+<em>i.gabor</em>
+The standard deviation of the Gaussian kernel before it is oscillated is tied to
+the user-specified window size.
+<p>
+The Gabor filter has both a real and imaginary component which can be used for
+different purposes.
+While <em>i.gabor</em> computes the real component by default, the imaginary
+component can be computed by using the flag <b>-i</b>.
+
+<p>
+By default <em>i.gabor</em> outputs each Gabor filter convolved raster map as
+individual raster maps
+with each part corresponding to its specified value.
+The <b>-c</b> flag can be used to create a singular combined raster map as the
+output.
+<p>
+Additionally, <em>i.gabor</em> offers statistical thresholding which will set
+values below a user-specified percentile (<b>threshold</b>) of the convolved
+image to 0 while keeping all values above the percentile the same.
+In conjunction with thresholding, a bitwise quantification (<b>-q</b>) method
+is implemented for the orientations specified by the user.
+
+<h2>NOTES</h2>
+As an orientation sensitive filter, the Gabor filter is unique in its
```

Comment:
Absolutely agreed with @ninsbl. Like I mentioned earlier personally, explaining why this module is important is key. Otherwise, you're limiting the user base to only those who know what the Gabor filter is and why/how it is useful. As is, the manual explains the technical aspects of Gabor, but it's not clear how it can be useful beyond creating directional texture rasters as shown in the examples. I mean, now what to do with the outputs? Making sense of the outputs and how to use them would be a great addition to the manual.